lib/rails/auth/x509/filter/pem.rb

```diff
@@ -5,7 +5,7 @@ module Filter
         # Extract OpenSSL::X509::Certificates from Privacy Enhanced Mail (PEM) certificates
         class Pem
           def call(pem)
-            OpenSSL::X509::Certificate.new(pem).freeze
+            OpenSSL::X509::Certificate.new(pem.delete("\t")).freeze
```

Comment:
Were you actually running into trouble with the PEM parser rejecting certs containing `"\t"`?